examples/lzapp-migration/tasks/solana/createOFT.ts

```diff
@@ -0,0 +1,327 @@
+import {
```

Comment:
Is this a pure copy/paste?

Reply:
we could maybe in a future PR add this as a task in `'@layerzerolabs/toolbox-hardhat'`